src/ImuSensor_TEST.cc

```diff
@@ -311,6 +311,83 @@ TEST(ImuSensor_TEST, ComputeNoise)
   EXPECT_GT(sensor->LinearAcceleration().SquaredLength(), 0.0);
 }
 
+//////////////////////////////////////////////////
+TEST(ImuSensor_TEST, Orientation)
+{
+  // Create a sensor manager
+  ignition::sensors::Manager mgr;
+
+  sdf::ElementPtr imuSDF, imuSDF_truth;
+
+  {
+    const std::string name = "TestImu_Truth";
+    const std::string topic = "/ignition/sensors/test/imu_truth";
+    const double update_rate = 100;
+    const auto accelNoise = noNoiseParameters(update_rate, 0.0);
+    const auto gyroNoise = noNoiseParameters(update_rate, 0.0);
+    const bool always_on = 1;
+    const bool visualize = 1;
+
+    imuSDF_truth = ImuSensorToSDF(name, update_rate, topic,
+      accelNoise, gyroNoise, always_on, visualize);
+  }
+
+  // Create an ImuSensor
+  auto sensor_truth = mgr.CreateSensor<ignition::sensors::ImuSensor>(
```

Comment:
```suggestion
  auto sensorTruth = mgr.CreateSensor<ignition::sensors::ImuSensor>(
```

Style nit. There are a few others too.

Reply:
fixed. a71f687